### ISSUE TITLE
popover offset not working 

### ISSUE THREAD

**Issue by ollixx** [bootstrap-vue/bootstrap-vue#7012]
### Offset Parameter  in popover has no effect

The offset parameter as described in the popover directive seems to have no effect.
Neither a number, a string, a function or an array (as documented in poppers.js) seem to have any effect.

### Steps to reproduce the bug

1. Go to https://bootstrap-vue.org/docs/directives/popover#positioning
2. change the code in one of the examples to use any of the documented offset versions
3. Check the result triggering the modified popover
4. See, that the popover is no rendered with an offset 

I used the directive in my code using an options object with offset property set to a function. It seems, that the function is never called.
see fiddler below

### Expected behavior

I would expect the offset to work with either the documented parameters (number / string) or as documented in poppers.js like here: https://popper.js.org/docs/v2/modifiers/offset/

### Versions

**Libraries:**

- BootstrapVue: 2.22.0
- Bootstrap: 4.6.1
- Vue: 2.7.4

**Environment:**

- Device: Mac Book Pro 2017
- OS: macOS 10.15.7
- Browser: Chrome
- Version: Version 103.0.5060.53 

### Demo link

https://jsfiddle.net/ollix/d03at81e/9/

**Comment by sookoll** [comment]
Yep, can confirm. Offset has not effect what so ever. Had to add custom class and set css position with !important to get it work.
